pkg/utils/config.go

```diff
@@ -0,0 +1,164 @@
+package utils
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/satori/go.uuid"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"os"
+	"os/user"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"text/template"
+)
+
+var defaultEnvConfig = `#generated by eden
+adam-dist: {{ .Dist }}/adam
+image-dist: {{ .Dist }}/images
+downloader-dist: {{ .Dist }}/eve/dist/{{ .Arch }}
+downloader-save: true
+eserver-port: 8888
+eserver-pid: {{ .Dist }}/eserver.pid
+eserver-log: {{ .Dist }}/eserver.log
+eve-arch: {{ .Arch }}
+eve-os: {{ .OS }}
+eve-accel: true
+adam-port: 3333
+eve-serial: 31415926
+eve-cert: {{ .Dist }}/certs/onboard.cert.pem
+eve-config: {{ .Dist }}/qemu.conf
```

Comment:
eve-config is super overloaded -- lets rename it to qemu-config

Also, please see my comment on the hierarchical configs -- this is one of the places where I'd love to have the default qemu-config sit in $HOME/.eden as well since it is unlikely to change much on a given system

Reply:
Done:
https://github.com/lf-edge/eden/blob/ea676f293e39ed98b8fd87103d1a4880316554e5/pkg/utils/config.go#L86-L89